masschallenge/impact-api
# AC-5147: Add "desired" events are to user histories

Work done:
- Added 'became desired judge' and 'became desired mentor' events
- Refactoring to keep things DRY

To test:
- Visit the history page for an expert, e.g., http://localhost:8000/api/v1/user/182/history/
- See the desired events (if appropriate)
Note: The "latest_datetime" is expected to be June 6, 2017 for events that occurred before that date since we don't have creation information for Program Role Grants.
